packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts

```diff
@@ -28,6 +28,15 @@ export const erc20FeeProxyArtifact = new ContractArtifact<ERC20FeeProxy>(
           address: '0x131eb294E3803F23dc2882AB795631A12D1d8929',
           creationBlockNumber: 13127007,
         },
+        alfajores: {
```

Comment:
Sorry for the late review.
Is it normal that the network 'celo' is not given here?
@bertux

Reply:
@vrolland , it was added later on, in a separate PR